### ISSUE TITLE
Use BoardGameGeek for game source

### ISSUE THREAD

**Issue by jgimbel** [game-curator/game-curator#1]
Use bgg to query board game geek for board games when creating the booklet. For short term maybe just a get request and pull the first result. Long term do type ahead lookups.
[Link to npm package](https://www.npmjs.com/package/bgg)

**Comment by liamdawson** [comment]
Integration with BGG is one of the ideas I want to explore, what sort of integrations would you be looking for? I don't want to pull the blurbs from BGG wholesale, as they're not necessarily consistent/all accessible to newer players, but building a booklet for your collection is something I want to explore.

**Comment by liamdawson** [comment]
Blocked by #4, for now.

**Comment by liamdawson** [comment]
With new changes, I'll be integrating (in some ways) with BGG, but I have no plans to pull game information from the site. I'll explore the integrations I plan for BGG in another issue.
